people/yumao/page.en.md

```diff
@@ -0,0 +1,152 @@
+---
+name: Yumao
+info:
+    alias: Yumao, Zhangqianye, Yunyu
+    location: Shanghai
+---
+
+> "I will transform myself into gentle breeze and rain, then surround you all along."
+>
+> By Yumao
+>
+> "It feels like that you are hiding in crystal alone, all I can see is your shattered figure. "
+>
+> By Neko
+
+## Description
+
+Yumao (Zhangqianye) is a trans girl, who is gentle, smart, tough and lovely.
+
+She introduced herself in her profile:
+Perhaps Yumao doesn't like taking part in group activities.
+I love white and red, as well as that kind of rainbow in Glitch.
+And I have a sweet tooth, I will feel happy for a long time while holding a packet of white rabbit creamy candies.
```

Comment:
I have a sweet tooth, I will feel happy for a long time after holding a packet of white rabbit creamy candies in my hands.